### DISCUSSION TITLE
Is it possible to create Organization Chart (Org Chart) like this one, with Image Icons etc.

### DISCUSSION THREAD

**Discussion by webdada** [xyflow/xyflow#4474] [General]
Like this one:
https://www.jointjs.com/demos/organizational-chart

The goal is to visualize close relationships in a social network, like first-degree relationship, and second-degree relationship and so on. 

I could not find a demo of xyflow for Org Charts, but other JS libraries like JointJS have a good example.

**Comment by bcakmakoglu** [comment]
Yes, building a chart like that is possible.
There are some layouting examples that use a layouting algo to create charts like that, [like the dagre example.](https://reactflow.dev/examples/layout/dagre)
Consider that XYFlow is giving you the tools to implement these use-cases yourself unlike other libraries that focus on solving these sort of specific use-cases for you.

Using custom nodes, which ultimately are just `<div>` elements, you can implement whatever sort of styling and behavior you want, including images and whatnot.

**Reply by CSworenI** [reply]
I actually found the [Elkjs Tree](https://reactflow.dev/examples/layout/elkjs) example much more useful. Combine that with the `onNodeClick` ReactFlow attribute and a modal dialog, with some styling it should be super easy to replicate that functionality.
